plugins/carbon_black_defense/unit_tests/test_find_event.py

```diff
@@ -0,0 +1,41 @@
+from unittest import TestCase
+from komand_carbon_black_defense.connection.connection import Connection
+from komand_carbon_black_defense.actions.find_event import FindEvent
+from insightconnect_plugin_runtime.exceptions import PluginException
+import json
+import logging
+import sys
+import os
+
+sys.path.append(os.path.abspath("../tests/"))
+
+
+class TestFindEvent(TestCase):
+    def test_integration_find_event(self):
+
+        log = logging.getLogger("Test")
+        test_conn = Connection()
+        test_action = FindEvent()
+
+        test_conn.logger = log
+        test_action.logger = log
+
+        try:
+            with open("../tests/find_event.json", encoding="utf-8") as file:
+                test_json = json.loads(file.read()).get("body")
+                connection_params = test_json.get("connection")
+                action_params = test_json.get("input")
+        except PluginException:
+            message = """
```

Comment:
Unfortunately these files can't really be uploaded as part of the PR. Its a great debugging tool, but the ../tests/*.json files won't (and shouldn't) be uploaded with the PR. The Unit Test check on the PR is only passing because the folder you placed these in are called unit_tests rather than unit_test.
If you want tests that can be used for regression testing, you'll need to follow a different structure that involves "mocking" the requests

Reply:
Uploaded a test file and json.resp files for the TestFindEvent class